goon/browserassets/css/browserOutput-dark.css

```
@@ -58,11 +58,16 @@ h1.alert, h2.alert			{color: #FFF;}
 .kidan					{color: #C64C05;}
 .drask					{color: #a3d4eb;}
 .shadowling				{color: #Cb2799;}
-.cultspeech				{color: #AF0000;}
-.cultitalic				{color: #A60000;}
-.cultlarge				{color: #A60000;}
-.narsie					{color: #A60000;}
-.narsiesmall			{color: #A60000;}
+.cultspeech				{color: #AF0000; font-style: italic;}
+.cultitalic				{color: #A60000; font-style: italic;}
+.cultlarge				{color: #A60000; font-weight: bold; font-size: 120%;}
+.narsie					{color: #A60000; font-weight: bold; font-size: 300%;}
+.narsiesmall			{color: #A60000; font-weight: bold; font-size: 200%;}
+.clock					{color: #e6a400; font-weight: bold; font-style: italic;}
+.clockspeech			{color: #cc9200; font-style: italic;}
```

Относительно `browserOutput.css` тут меняется только цвет.
Не нужно дублировать не меняющиеся правила.
```suggestion
.clockspeech			{color: #cc9200;}
```